include/aspect/material_model/interface.h

```diff
@@ -785,6 +785,14 @@ namespace aspect
         * transport) in each quadrature point.
         */
         std::vector<double> rhs_melt_pc;
+
+        /**
+         * Force tensor (elastic terms) on the right-hand side for the conservation of
+         * momentum equation (first part of the Stokes equation) in each
+         * quadrature point.
+         */
+        std::vector<Tensor<2,dim> > rhs_e;
```

Comment:
By putting these in here you tie them very closely to the other `AdditionalStokesRHS` outputs. I could also imagine moving the `ElasticAdditionalOutputs` here, and placing `rhs_e` into `ElasticAdditionalOutputs`. I think this would make the logic in the assembler simpler. Just a suggestion though, if you prefer it this way I am fine with it.